csrc/device_lower/validation.cpp

```diff
@@ -47,7 +47,10 @@ class ValidateSiblings : public IterVisitor {
   using IterVisitor::handle;
 
   void dispatch(Expr* expr) final {
-    if (!ir_utils::isTvOp(expr) || expr->outputs().size() < 2) {
+    // Skip BlockQuantization.
+    // It has sibling outputs which differ from each other
+    if (!ir_utils::isTvOp(expr) || expr->outputs().size() < 2 ||
+        expr->isA<BlockQuantizationOp>()) {
```

Comment:
```suggestion
        !ir_utils::hasUniformSiblings(expr)) {
```

And update `hasUniformSiblings`